src/Pages/Auth/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,169 @@
+import { background5 } from "@/Assets/Images";
+import { ChangeData } from "@/Redux/Featuers/Auth/ChangeSlice";
+import useAction from "@/Utils/Hooks/UseAction";
+import { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { FaCheckCircle, FaKey } from "react-icons/fa";
+import { FaRegCircleXmark } from "react-icons/fa6";
+import { TbFidgetSpinner } from "react-icons/tb";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+
+export default function ChangePassword() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [showPass, setShowPass] = useState(false);
```

Comment:
`showPass` makes the state variable seem like a function not a state, we can change it to something like `isPasswordShown` to indicate that it is a state variable holding a boolean value.